rero_ils/modules/cli.py

```diff
@@ -1178,6 +1212,49 @@ def reindex(pid_type, no_info):
         )
 
 
+@utils.command('reindex_missing')
+@click.option('-t', '--pid-type', multiple=True, required=True)
+@click.option('-v', '--verbose', 'verbose', is_flag=True, default=False)
+@with_appcontext
+def reindex_missing(pid_type, verbose):
```

Comment:
cosmetic : maybe `pid_types` name would be better if this argument is an array.

Reply:
ok